### DISCUSSION TITLE
Comment field continues to open and steal focus

### DISCUSSION THREAD

**Discussion by iGEL** [community/community#74790] [Pull Requests]
### Select Topic Area

Bug

### Body

Hi there!

I sometimes hit a bug where a comment field on a PR opens all the time and steals the focus from wherever I just had it on the PR. It doesn't matter whether I click close on the comment or not, reload the tab or reopen the browser.

This is super annoying and distracting.

I'm using Firefox 119.0.1 (64-bit) on Ubuntu 23.10/Sway with these add-ons:
![image](https://github.com/community/community/assets/36442/5a9dda09-76fd-4018-be15-919cdf38cab7)

But this was happening before with older FF/Ubuntu versions and without the first 3 addons.

**Comment by svenja-haas** [comment]
this happens to me so much and is really frustrating :/
